packages/bundle-size/src/util.ts

```diff
@@ -17,26 +17,54 @@ import { buildSync } from "esbuild";
 import { sizes } from "./constants";
 import { readFileSync, writeFileSync } from "node:fs";
 
-type MarkdownRow = {
+type Row = {
   name: string;
   files: number;
   size: number;
   minified: number;
   compressed: number;
 };
 
-export function generateMarkdownTable(rows: MarkdownRow[]) {
-  let md = `| code generator  | files    | bundle size             | minified               | compressed         |\n`;
-  md += `|-----------------|----------|------------------------:|-----------------------:|-------------------:|\n`;
-  for (const row of rows) {
-    md += `| ${row.name} | ${row.files} | ${formatBytes(row.size)} | ${formatBytes(row.minified)} | ${formatBytes(row.compressed)} |\n`;
+export function generateMarkdownTable(rows: Row[]) {
```

Comment:
Since prettier now runs on markdown, I've updated this code to generate formatted markdown, so that we don't get flip-flopping state.